<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(34, 45, 50);
border-width: 2px;
border-style: solid;
border-color: #222d31;</string>
     </property>
     <property name="text">
      <string> Analizador léxico y sintáctico para Kratos</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>12</pointsize>
               <fontweight>Light</fontweight>
              </font>
             </property>
             <property name="text">
              <string>Editor de código</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="archivo_abierto">
             <property name="font">
              <font>
               <family>Bahnschrift</family>
              </font>
             </property>
             <property name="text">
              <string>Ningún archivo abierto</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="icono_correcto">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
             <property name="indent">
              <number>-1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnCopiar">
           <property name="toolTip">
            <string>Copiar</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPegar">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCortar">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="archivo_guardado">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
            </font>
           </property>
           <property name="text">
            <string>No guardado</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnAbrir">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
            </font>
           </property>
           <property name="text">
            <string>Abrir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLimpiar">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
            </font>
           </property>
           <property name="text">
            <string>Limpiar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGuardar">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
            </font>
           </property>
           <property name="text">
            <string>Guardar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAnalizar">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
            </font>
           </property>
           <property name="text">
            <string>Analizar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>420</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSalir">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
            </font>
           </property>
           <property name="text">
            <string>Salir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>12</pointsize>
           <fontweight>Light</fontweight>
          </font>
         </property>
         <property name="text">
          <string>Tokens</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblTokens">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tokens encontrados:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listTokens">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>12</pointsize>
           <fontweight>Light</fontweight>
          </font>
         </property>
         <property name="text">
          <string>Errores léxicos y/o sintácticos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblErrores">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Errores encontrados: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listErrores">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
